samples/drivers/espi/Kconfig

```diff
@@ -26,18 +26,30 @@ config ESPI_GPIO_DEV_NEEDED
 
 if ESPI_GPIO_DEV_NEEDED
 
-config ESPI_GPIO_DEV
-	string "Name of the GPIO port"
+config ESPI_GPIO_DEV0
+	string "Name of the GPIO port 1"
 	default "GPIO_0"
```

Comment:
port 1 is called GPIO_0?

Reply:
Corrected KConfig port names and rebased the branch